app.js

```diff
@@ -394,11 +394,11 @@ app.get("/deleteRepository", (req, res)=>{
     .catch((err)=>{console.log(err)});
  });
 
- app.get("/newRoute", (req, res){
+ app.get("/newRoute", (req, res)=>{
 
-    // New route added for Learning PRs.
+    res.render('newRoute');

```

Comment:
Ok

Reply:
https://github.com/usamaahmed97/practice-app/issues/3

This issue has been fixed. Created a newRoute ejs page and used it in GET request for proper routing.